requirements.txt

```diff
@@ -7,15 +7,18 @@
 argparse==1.4.0           # via ofs
 babel==0.9.6
 beaker==1.7.0
+bleach==1.4.2
 decorator==4.0.4          # via pylons, sqlalchemy-migrate
 fanstatic==0.12
 formencode==1.3.0         # via pylons
 Genshi==0.6
 Jinja2==2.6
 mako==1.0.2               # via pylons
+Markdown==2.1
```

Comment:
Claimed to, didn't work for me.  Perhaps i got myself muddled with the missing ordereddict.  Will try 2.4 again, although we should remember to update to 2.6.5 when we drop Py26 support.